pkg/virt-controller/services/vm.go

```diff
@@ -48,8 +48,8 @@ type VMService interface {
 	GetRunningMigrationPods(*corev1.Migration) (*v1.PodList, error)
 	CreateMigrationTargetPod(migration *corev1.Migration, vm *corev1.VM) error
 	UpdateMigration(migration *corev1.Migration) error
-	FetchVM(vmName string) (*corev1.VM, bool, error)
-	FetchMigration(migrationName string) (*corev1.Migration, bool, error)
+	FetchVM(vmName string, namespace string) (*corev1.VM, bool, error)
+	FetchMigration(migrationName string, namespace string) (*corev1.Migration, bool, error)
```

Comment:
Here we don't have a KubevirtClient for yet, so fine for me if you keep it for now. WIll provide a PR which will give us such a client for Migrations too.